integrations/terraform/tfschema/custom_types.go

```diff
@@ -39,7 +39,16 @@ func GenSchemaBoolOption(_ context.Context, attr tfsdk.Attribute) tfsdk.Attribut
 	}
 }
 
-// GenSchemaBoolOptions returns Terraform schema for Traits type
+// GenSchemaBoolOptions returns Terraform schema for NullBoolOption type
+func GenSchemaNullBoolOption(_ context.Context, attr tfsdk.Attribute) tfsdk.Attribute {
+	return tfsdk.Attribute{
+		Optional:    true,
+		Type:        types.BoolType,
+		Description: attr.Description,
+	}
+}
```

Comment:
There's a similar type in `tfschema/token/custom_types.go`, can you align the name and the code so we don't have two types doing the same thing?